RELEASES.md

```diff
@@ -0,0 +1,25 @@
+# Release Automation 🤖
```

Comment:
this is a copy of [release-automation](https://github.com/valora-inc/release-automation/blob/main/README.md) README